feelslikemonday/app/src/main/java/com/example/feelslikemonday/model/EmotionBookAdapter.java

```diff
@@ -0,0 +1,50 @@
+package com.example.feelslikemonday.model;
+
+import android.content.Context;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ArrayAdapter;
+import android.widget.TextView;
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import com.example.feelslikemonday.R;
+
+
+import java.util.ArrayList;
+
+
+public class EmotionBookAdapter extends ArrayAdapter<MoodEvent> {
+
+    private Context context;
+    private int mResource;
+    // constructor
+    public EmotionBookAdapter(Context context, int resource, ArrayList<MoodEvent> objects) {
+        super(context, resource, objects);
+        this.context = context;
+        mResource = resource;
+    }
+
+    @NonNull
+    @Override
+    public View getView(int position, @Nullable View convertView, @NonNull ViewGroup parent) {
+
+        String var1_date = getItem(position).getDate().toString();//??-----
```

Comment:
We can treat the date as String for now.

Reply:
noted